python/doc/examples/calibration/bayesian_calibration/plot_bayesian_calibration.py

```diff
@@ -149,7 +149,7 @@
     y_obs[i, 0] = linkFunction(thetaTrue)[0] + noiseSample[i, 0]
```

Comment:
Ok. I fully agree (I did not do that prior to now because I did not know about the draft feature before 1 year: now that draft PR can be created, we can benefit from CI event for our own developments). Fixed.